### DISCUSSION TITLE
add baseurl error

### DISCUSSION THREAD

**Discussion by LEESHOHYUN** [mmistakes/minimal-mistakes#3110] [Q&A] [answered]
https://github.com/LEESHOHYUN/LEESHOHYUN.github.io

above, this is my github page, and i change _config.yml file, 
baseurl                  : "/blog"
to add /blog to my site.
and I tried `bundle exec jekyll serve` , and 127.0.0.1:4000/blog is successfully created
but I tried to username.github.io/blog , and it doesn't work, only 404 file not found is shown
someone help me to solve this problem?

**Comment by iBug** [comment, marked as answer]
Use of `baseurl` is not supported on GitHub Pages. The URL is determined from the repository name, so the repository `<username>.github.io` will always be published at `/` and the repository `example` will always be published at `/example`.

<https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages>

**Reply by LEESHOHYUN** [reply]
oh...I understand. then if i change my repository name (remove .github.io), may I use baseurl? or if i want to use baseurl, what should I do? I read installation page <https://mmistakes.github.io/minimal-mistakes/docs/installation/> and quickstart page, but don't know what way to use.

**Reply by iBug** [reply]
Name your repo exactly `blog`.

**Reply by LEESHOHYUN** [reply]
oh okay thanks!
